build/media_source/com_finder/js/finder.es6.js

```diff
@@ -52,6 +52,14 @@
     }
   };
 
+  // Submits the form programmatically
+  const submitForm = event => {
+    const form = event.target.closest('form');
+    if(form) {
```

Comment:
```suggestion
  const submitForm = (event) => {
    const form = event.target.closest('form');
    if (form) {
```
Fix code style errors reported by drone